buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestIntegTestTask.groovy

```diff
@@ -215,7 +215,7 @@ public class RestIntegTestTask extends DefaultTask {
      * @param project The project to add the copy task to
      * @param includePackagedTests true if the packaged tests should be copied, false otherwise
      */
-    static Task createCopyRestSpecTask(Project project, Provider<Boolean> includePackagedTests) {
+    Task createCopyRestSpecTask() {
```

Comment:
The static method was called with the value of `includePackagedTests` in what happened to be the right order so that changes to the property were picked up in time and the right value was passed into the `project.afterEvaluate` ( which had no use, since the value was already picked up anyhow ). 
This can't be static to make changes to the property reflected as they should. Gradle 5 seems to have changed the order in which some operations are executed.  I wish they would have an option to randomize ordering of otherwise undefined sequences.